frontend/public/actions/k8s.ts

```diff
@@ -12,12 +12,16 @@ import { makeReduxID } from '../components/utils/k8s-watcher';
 import { APIServiceModel } from '../models';
 import { coFetchJSON } from '../co-fetch';
 import { referenceForModel, K8sResourceKind, K8sKind } from '../module/k8s';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src/models';
```

Comment:
We should never have a import from core console to a plugin (operator-lifecycle-manager). We either need to move the model definition into core console or create a plugin extension point for managed by links. I would just move the model into core for now.

Reply:
Good to know; will fix!